### PR TITLE
SEO Optimizations

### PR DESCRIPTION
**Pull request by Logan-Ruf**

### CODE REVIEW

File: __templates/post.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title:  Blog Post Title
+description: This description describes what the post is about
+author: author
+image: assets/blog/2020-2-5-tailwindcss/4wsata.jpg
+categories: blog
```
Comment (ckcollab) [inline __templates/post.md:6]: tyvm!

File: _config.yml [hunk 2]
```diff
@@ -21,6 +19,29 @@ collections:
...
+image: assets/images/ckclogo.png
+
+social:
+  name: CK Collab
+  links:
+    - https://www.facebook.com/ckcollab/
+    - https://twitter.com/ckcollab/
+    - https://www.linkedin.com/company/33456525/
+    - https://github.com/orgs/ckc-org/
+    - https://clutch.co/profile/ckc
```
Comment (ckcollab) [inline _config.yml:42]: aaaaaaawesome!

**Review by ckcollab — APPROVED** [review]
hmm.. probably not an easy way to test these OpenGraph thingies, I'm way down with pushing many commits to master here to figure this out!

File: __templates/post.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+title:  Blog Post Title
+description: This description describes what the post is about
+author: author
+image: assets/blog/2020-2-5-tailwindcss/4wsata.jpg
```
Comment (Logan-Ruf) [inline __templates/post.md]:
```suggestion
image: assets/blog/2020-1-1-blog-post-name/image.jpg
```

**Pull request merged**